package.json

```
@@ -10,7 +10,9 @@
 		"Other"
 	],
 	"activationEvents": [
-		"onStartupFinished"
+		"onStartupFinished",
+		"onCommand:Spotify.Pause",
+		"onCommand:Spotify.Play"
```

```suggestion
		"onStartupFinished"
```

Extensions are activated only once and the `onStartupFinished` event will run before any command, so adding commands as activation events doesn't currently do anything. In the future that might change, since activating on startup is kind of a hack and won't be the best user experience in the long run...but we have other things to build out first.

The commands listed here don't exist yet, since we've been using the `spotifymlh.` prefix for everything. I'm a fan of moving over to just `spotify.`, but that hasn't happened yet so these command activation events are extra not doing anything right now.